src/bin/tikv-server.rs

```diff
@@ -198,6 +198,14 @@ fn get_rocksdb_option(matches: &Matches, config: &toml::Value) -> RocksdbOptions
     };
     opts.set_min_write_buffer_number_to_merge(min_write_buffer_number_to_merge as i32);
 
+    let max_background_threads = get_integer_value("",
+                                                   "rocksdb.max-background-threads",
+                                                   matches,
+                                                   config,
+                                                   Some(3),
```

Comment:
Some(1) maybe better, because just when system is bottlenecked by RocksDB and then should up this number. thanks.